actions/apiCitiesAction.js

```diff
@@ -0,0 +1,13 @@
+export const addCitiesToStore = 'addCitiesToStore';
```

Comment:
Move these constants to seperate folder with name actionConstants.

in that folder define it as objects.

e.g:

export default {
getSportsList: 'GET_SPORTS_LIST';
}